src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIOTalonFX.java

```diff
@@ -143,42 +167,60 @@ public void updateInputs(ShooterIOInputs inputs) {
     acceleratorDisconnectedAlert.set(!acceleratorConnected);
 
     inputs.leftPosition = Rotation2d.fromRotations(leftPosition.getValueAsDouble());
+    inputs.leftVelocitySetpointRadiansPerSecond =
+        Units.rotationsToRadians(leftVelocitySetpointRadiansPerSecond.getValueAsDouble());
+
+    inputs.leftVelocityErrorRadiansPerSecond =
+        Units.rotationsToRadians(leftVelocityErrorRadiansPerSecond.getValueAsDouble());
     inputs.leftVelocityRadiansPerSecond =
-        Units.rotationsToRadians(leftVelocityRadiansPerSecond.getValueAsDouble());
+        Units.rotationsToRadians(leftVelocityRotationsPerSecond.getValueAsDouble());
     inputs.leftAppliedVolts = leftAppliedVolts.getValueAsDouble();
     inputs.leftCurrentAmps = leftCurrentAmps.getValueAsDouble();
     inputs.leftTemperatureCelsius = leftTemperatureCelsius.getValueAsDouble();
 
     inputs.rightPosition = Rotation2d.fromRotations(rightPosition.getValueAsDouble());
+    inputs.rightVelocitySetpointRadiansPerSecond =
+        Units.rotationsToRadians(rightVelocitySetpointRadiansPerSecond.getValueAsDouble());
+    inputs.rightVelocityErrorRadiansPerSecond =
+        Units.rotationsToRadians(rightVelocityErrorRadiansPerSecond.getValueAsDouble());
     inputs.rightVelocityRadiansPerSecond =
-        Units.rotationsToRadians(rightVelocityRadiansPerSecond.getValueAsDouble());
+        Units.rotationsToRadians(rightVelocityRotationsPerSecond.getValueAsDouble());
     inputs.rightAppliedVolts = rightAppliedVolts.getValueAsDouble();
     inputs.rightCurrentAmps = rightCurrentAmps.getValueAsDouble();
     inputs.rightTemperatureCelsius = rightTemperatureCelsius.getValueAsDouble();
 
     inputs.acceleratorPosition = Rotation2d.fromRotations(acceleratorPosition.getValueAsDouble());
     inputs.acceleratorVelocityRadiansPerSecond =
-        Units.rotationsToRadians(acceleratorVelocityRadiansPerSecond.getValueAsDouble());
+        Units.rotationsToRadians(acceleratorVelocityRotationsPerSecond.getValueAsDouble());
     inputs.acceleratorAppliedVolts = acceleratorAppliedVolts.getValueAsDouble();
     inputs.acceleratorCurrentAmps = acceleratorCurrentAmps.getValueAsDouble();
     inputs.acceleratorTemperatureCelsius = acceleratorTemperatureCelsius.getValueAsDouble();
```

Comment:
Make sure to update the inputs in the same order that the inputs are created in ShooterIO.java